lab-ben/lib/greet.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+module.exports = exports = {};
+
+
+exports.say = function(name) {
+  return `hello ${name}`;
+};
+
+if(process.argv[2]) {
```

Comment:
**TODO FOR LOST POINTS**

Great job on getting the bonus! The only reason I'm taking off 1 point is because the invocation of this functions should be happening in ``index.js``.  Your bonus point will make up for it though.